src/components/CommentCard.jsx

```diff
@@ -0,0 +1,24 @@
+import "../styles/comment-card.css";
+
+const CommentCard = ({ comment }) => {
```

Comment:
this also isn't for this ticket, I know you dont want to remove the code but you could've just not committed it for this pull request